install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -464,7 +464,7 @@ data:
 
   # A space separated list of iptables chains to disable when installing feeder rules.
   disable-iptables-feeder-rules: {{ .Values.global.disableIptablesFeederRules | join " " | quote }}
-{{ if .Values.global.ipam }}
+{{ if and .Values.global.ipam (not (or .Values.global.gke.enabled ( or .Values.global.eni .Values.global.azure.enabled ))) }}
```

Comment:
Why this change? Doesn't this lead to a secondary `ipam:` line? Both Azure and ENI have a separate section which also sets `ipam:`. I'm fine with this but then you have to remove that other one.

Reply:
@tgraf I think you missed the "not"? This commit is fixing the "Both Azure and ENI have a separate section which also sets ipam:" since that's the current behavior.